INSTALL.md

```diff
@@ -1,10 +1,11 @@
 # Install Zync
-
-Download this repository and run `bundle install`.
```

Comment:
What does `this` refer to?

Reply:
@porueesq You are reviewing an old version 😄 This is not here anymore 👍